Data/Helpers/UserHelper.cs

```diff
@@ -17,4 +17,11 @@ public static bool IsStudent(this ClaimsPrincipal user)
         if (claim == null) return false;
         return claim.Value is "0" or "Student";
     }
+
+    public static bool IsModerator(this ClaimsPrincipal user)
+    {
+        Claim claim = user.Claims.SingleOrDefault(e => e.Type == "UserType");
+        if (claim == null) return false;
+        return claim.Value is "1" or "Moderator";
```

Comment:
1 should be 4

Reply:
This isn't the case in https://github.com/team-jasmijn/backend/blob/features/list-companies/Data/Helpers/UserHelper.cs#L18 though. Since Student is 2, not 0